docs/src/configuration/services/redis.md

```diff
@@ -40,16 +40,16 @@ Platform.sh supports two different Redis configurations: One persistent (useful
 Versions 3.0 and higher support up to 64 different databases per instance of the service, but Redis 2.8 is configured to support only a single database.
 {{< /note >}}
 
-## Ephemeral Redis
+## Service types
 
-The `redis` service type is configured to serve as an LRU cache; its storage isn't persistent.
-It is not suitable for use except as a disposable cache.
+There are two types of the Redis service depending on what you want to do with it.
 
-To add an ephemeral Redis service, specify it like so:
+### Ephemeral Redis
 
-{{< readFile file="src/registry/images/examples/full/redis.services.yaml" highlight="yaml" location=".platform/services.yaml" >}}
+The ephemeral Redis service is configured to serve as an LRU cache; its storage isn't persistent.
```

Comment:
Since LRU is not the only type of behavior that can be expected (LFU, ...), we could also change the sentence to:
```suggestion
The ephemeral Redis service is configured to serve as a cache; its storage isn't persistent.
```
and remove:
> It isn't suitable for use except as a disposable cache.